tuplex/core/include/ee/local/LocalBackend.h

```diff
@@ -50,6 +51,8 @@ namespace tuplex {
         std::vector<Executor*> _executors; //! drivers to be used
         std::unique_ptr<JITCompiler> _compiler;
 
+        void executeSortStage(SortStage* sstage);
```

Comment:
add doxygen comment on what this does (i.e. some details on predecessor stages etc.)

Reply:
Done.